app/src/main/java/com/droidhats/campuscompass/viewmodels/FavoritesViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.droidhats.campuscompass.viewmodels
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import com.droidhats.campuscompass.models.FavoritePlace
+import com.droidhats.campuscompass.roomdb.FavoritesDatabase
+import com.google.android.gms.maps.model.LatLng
+import kotlin.math.sin
+import kotlin.math.sqrt
+import kotlin.math.cos
+import kotlin.math.asin
+import kotlin.math.pow
+
+/*
+  This class must extend AndroidViewModel instead of just ViewModel because
+  ContentResolver requires the application context to be able to query calendar info
```

Comment:
Update this line comment with "ContentResolver requires the application context to be able to query **favorite place info**"